tools/Azure.Mcp.Tools.AzureManagedLustre/src/Commands/FileSystem/FileSystemGetSkuInfoCommand.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Mcp.Core.Commands;
+using Azure.Mcp.Tools.AzureManagedLustre.Options;
+using Azure.Mcp.Tools.AzureManagedLustre.Options.FileSystem;
+using Azure.Mcp.Tools.AzureManagedLustre.Services;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Mcp.Tools.AzureManagedLustre.Commands.FileSystem;
+
+public sealed class FileSystemGetSkuInfoCommand(ILogger<FileSystemGetSkuInfoCommand> logger)
+    : BaseAzureManagedLustreCommand<FileSystemGetSkuInfoOptions>(logger)
+{
+    private const string CommandTitle = "Get AMLFS SKU information";
+
+    public override string Name => "get-sku-info";
+
+    public override string Description =>
+        """
+        Retrieves available Azure Managed Lustre in all the regions or in a specific regions. Use to discover capabilities, SKU names and zone support.
+        """;
+
+    public override string Title => CommandTitle;
+
+    public override ToolMetadata Metadata => new() { Destructive = false, ReadOnly = true };
+
+    private static readonly Option<string> _locationOption = AzureManagedLustreOptionDefinitions.LocationOption;
+
+    protected override void RegisterOptions(Command command)
+    {
+        base.RegisterOptions(command);
+        _locationOption.IsRequired = false;
```

Comment:
Should an `OptionalLocationOption` be added instead of mutating an option which could cause issue